### ISSUE TITLE
seraphine's passive music

### ISSUE THREAD

**Issue by OuchyMan** [hydy100/R3nzSkin#252]
Seraphine's passive music does not change when switching skins. Is this fixable?

**Comment by mish20011** [comment]
> Seraphine's passive music does not change when switching skins. Is this fixable?

if you use cslolmanager you can try using this instead idk if it works though 
https://github.com/koobzaar/lol-skins

**Comment by mish20011** [comment]
> Seraphine's passive music does not change when switching skins. Is this fixable?

yeah i just tried it with custom skin instead and it works (the seraphine music), but some still don't (garen E)
